main.py

```diff
@@ -352,13 +475,18 @@ def sigint_handler(signum, frame):
         TargetHostTest('api.build02.gcp.ci.openshift.org', TestType.DNS_TCP_LOOKUP, None),
         TargetHostTest('api.build04.34d2.p2.openshiftapps.com', TestType.DNS_TCP_LOOKUP, None),
         TargetHostTest('static.redhat.com', TestType.DNS_TCP_LOOKUP, None),
+
+        TargetHostTest('', TestType.LIVENESS_PROBE_ICMP, None),
+        TargetHostTest('', TestType.LIVENESS_PROBE_UDP, None),
+        # TargetHostTest('', TestType.LIVENESS_PROBE_TCP, None),
```

Comment:
Why is TCP turned off?

Reply:
`SYN flood` :-D
This implementation does not actually establish the TCP connection. Similar as UDPping, it cares only if the remote host is alive, not actually communicating. We will use it later if we find evidence for SDN issues.